common4j/src/main/com/microsoft/identity/common/java/eststelemetry/TelemetryUtils.java

```diff
@@ -22,15 +22,16 @@
 // THE SOFTWARE.
 package com.microsoft.identity.common.java.eststelemetry;
 
-import lombok.NonNull;
-
-import com.microsoft.identity.common.java.util.StringUtil;
 import com.microsoft.identity.common.java.telemetry.TelemetryEventStrings;
+import com.microsoft.identity.common.java.util.StringUtil;
+
+import edu.umd.cs.findbugs.annotations.Nullable;
+import lombok.NonNull;
 
 public class TelemetryUtils {
 
-    static boolean getBooleanFromSchemaString(final String val) {
-        return val.equals(SchemaConstants.Value.TRUE);
+    static boolean getBooleanFromSchemaString(@Nullable final String val) {
```

Comment:
nit: SchemaString is not suppose to be null right? :)
Should we change the name of the method to getBooleanFromString.

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1799/commits/a33a919716d6442d626a702698c294a102673489